lightning-background-processor/src/lib.rs

```diff
@@ -151,7 +148,12 @@ impl BackgroundProcessor {
 	/// [`ChannelManager`]. See [`FilesystemPersister::persist_manager`] for Rust-Lightning's
 	/// provided implementation.
 	///
-	/// Typically, users should either implement [`ChannelManagerPersister`] to never return an
+	/// `persist_graph` is responsible for writing out the [`NetworkGraph`] to disk. See
```

Comment:
In the paragraphs above, I think the reference to `persist_manager` was at one point referring to a parameter to `start` but has been renamed a few times since. We should update the docs accordingly and use similar wording in this paragraph for `persist_graph` now that these are methods on `persister`.